source/MasterData.Domain/ProcessDelegations/ProcessDelegation.cs

```diff
@@ -63,7 +59,7 @@ private ProcessDelegation()
     /// <summary>
     /// The type of process that is delegated ex: PROCESS_REQUEST_ENERGY_RESULTS
```

Comment:
```suggestion
    /// The type of process that is delegated ex: PROCESS_REQUEST_ENERGY_RESULTS
```

^ lets remove this comment